### PR TITLE
Add `011-Vincent` notebook contents

### PR DESCRIPTION
**Pull request by Haleshot**
Add relevant notebooks shared:
- Seam-carving demo on HDF
- Interactive matrix visualizations in the gallery.
- Add and update relevant README contents

### CODE REVIEW

**Comment by Haleshot** [comment]
Let me know if the changes made are appropriate.

**Review by Haleshot — COMMENTED** [review]
Made it so that the latest month gets displayed in the root README (with months before that with toggles in them, displayed in desc order). Added relevant notebooks shared in the socials.

**Comment by Haleshot** [comment]
Just a _reminder_, @akshayka

File: README.md [hunk 1]
```diff
@@ -213,6 +237,6 @@ We love to share our community spotlights across various platforms! You can find
 
 - LinkedIn: [marimo LinkedIn](https://www.linkedin.com/company/marimo-io/)
 - Twitter: [@marimo_io](https://x.com/marimo_io)
-- Discord: [marimo community](https://discord.gg/JE7nhX6mD8)
+- Discord: [marimo community](https://discord.gg/QdpFxJWhyt?ref=spotlights)
```
Comment (mscolnick) [inline README.md]: you can change this to `https://marimo.io/discord?ref=spotlights`. in case the discord link changes again

Comment (Haleshot) [reply README.md]: Yeah, did that. Thanks for spotting that.

**Pull request merged**